src/M6Web/Bundle/LogBridgeBundle/Config/FilterParser.php

```diff
@@ -57,169 +44,99 @@ protected function createFilter($name)
         return new Filter($name);
     }
 
-    /**
-     * isRoute
-     *
-     * @param string $name Route name
-     *
-     * @return bool
-     */
-    protected function isRoute($name)
+    protected function isRoute(string $name): bool
     {
-        return $this->router->getRouteCollection()->get($name) ? true : false;
+        return $this->router->getRouteCollection()->get($name) !== null;
     }
 
-    /**
-     * parse
-     *
-     * @param string $name   name
-     * @param array  $config configuration
-     *
-     * @throws ParseException
-     *
-     * @internal param array $filterConfig
-     *
-     * @return Filter
-     */
-    public function parse($name, array $config)
+    public function parse(string $name, array $config): Filter
     {
-        $filter = $this->createFilter($name);
-
-        if (!array_key_exists('route', $config) || !array_key_exists('method', $config) || !array_key_exists('status', $config)) {
+        if (
+            !array_key_exists('route', $config) ||
+            !array_key_exists('method', $config) ||
+            !array_key_exists('status', $config)
+        ) {
```

Comment:
As we use $config['route'] and others below, I suggest this non-blocking suggestion:
```suggestion
        $route = $config['route'] ?? null;
        $method = $config['method'] ?? null;
        $status = $config['status'] ?? null;
        if (
            null === $route ||
            null === $method ||
            null === $status
        ) {
```
and replace $config['route'] by $route below

Reply:
this change would throw an exception if the keys exist with a `null` value

Reply:
This won't work because we're just checking that the key exist, even if it's null.
Given `$config['route'] = null` then `!array_key_exists('route', $config)` is `false` but `($config['route'] ?? null) === null` is `true`

Reply:
You're right. But we check in parseRoute if $route is not null.
It was a suggestion